packages/core/src/feature-app-manager.ts

```diff
@@ -12,13 +12,22 @@ import {isFeatureAppModule} from './internal/is-feature-app-module';
 
 export interface FeatureAppEnvironment<
   TConfig,
+  TInstanceConfig,
   TFeatureServices extends FeatureServices
 > {
   /**
-   * A Feature App config object that is provided by the integrator.
+   * A Feature App config object that is provided by the integrator. The same
+   * config object is used for all Feature App instances with the same ID, which
+   * is defined in their {@link FeatureAppDefinition}.
    */
   readonly config: TConfig;
 
+  /**
+   * An optional Feature App config object that is intended for a specific
+   * Feature App instance.
+   */
+  readonly instanceConfig?: TInstanceConfig;
```

Comment:
Please remove the optional flag.

Reply:
Because the key is always set and only the value might be undefined?

Reply:
yes